config.py

```diff
@@ -0,0 +1,10 @@
+#--------------------------------------------------------------------------------
+# These tokens are needed for user authentication.
+# Credentials can be generates via Twitter's Application Management:
+#       https://apps.twitter.com/app/new
+#--------------------------------------------------------------------------------
+
+consumer_key = "XxXxXxxXXXxxxxXXXxXX"
```

Comment:
Any ideas on how to manage secrets? We use LastPass at work to manage and share credentials.